services/user.py

```
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.base_user import AbstractBaseUser
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
+) -> User:
+    user = User.objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+    return user
+
+
+def get_user(user_id: int) -> AbstractBaseUser:
+    return get_user_model().objects.get(id=user_id)

```

The `get_user` function uses `get_user_model().objects.get(id=user_id)`, which will raise a `DoesNotExist` exception if the user is not found. Consider handling this exception to provide a more user-friendly error message.